docs/misc/contributing.rst

```diff
@@ -0,0 +1,133 @@
+==============================
+Contributing to sed
+==============================
+
+Welcome to the sed project, a collaboration of the Open Community of Multidimensional Photoemission Spectroscopy.
+
+Whether you are a beamline scientist hoping to create a loader for your data, or would like to add a new feature to the project, we welcome your contributions.
+
+This guide will walk you through the process of setting up your development environment, and the workflow for contributing to the project.
+
+.. contents::
+
+Getting Started
+===============
+
+1. **Clone the Repository:**
+
+   - If you are a member of the repository, clone the repository to your local machine:
+
+    .. code-block:: bash
+
+        git clone https://github.com/OpenCOMPES/sed.git
+
+   - If you are not a member of the repository, clone your fork of the repository to your local machine:
+
+    .. code-block:: bash
+
+        git clone https://github.com/yourusername/sed.git
+
+
+
+2. **Install Python and Poetry:**
+   - Ensure you have Python 3.8, 3.9, or 3.10 and poetry installed.
+
+    .. code-block:: bash
+
+        pip install pipx
+        pipx install poetry
```

Comment:
Is it necessary/advisable to use pipx here? Not sure I understand completely what this does.

Reply:
Just an alternate to install poetry. It's the recommended version by poetry now. I tried it and it worked for me, much simpler than the alternates.